pennylane/transforms/classical_jacobian.py

```diff
@@ -184,16 +184,6 @@ def _jacobian(*args, **kwargs):  # pylint: disable=unused-argument
             argnum = 0 if wrapper_argnum is None else wrapper_argnum
 
             def _jacobian(*args, **kwargs):
-                if trainable_only:

```

Comment:
confused how this worked before.. `trainable_only` doesn't seem to be defined in this scope

Reply:
It is a kwarg in the `classical_jacobian`, but yes the implementation was very tricky.